gnssanalysis/gn_io/igslog.py

```diff
@@ -101,6 +129,22 @@ def find_recent_logs(logs_glob_path: str, rnx_glob_path=None) -> _pd.DataFrame:
     return recent_logs_df
 
 
+def determine_log_version(data: bytes):
+    REGEX_VERSION_1 = _re.compile(rb"""(site log\))""")
+    REGEX_VERSION_2 = _re.compile(rb"""(site log v2)""")
+
+    # check for version 1:
+    result_v1 = REGEX_VERSION_1.search(data)
```

Comment:
I suppose if version 1 is successful we don't need to check if version 2 is good?

Reply:
Yep true. This has been updated so that a check of both is no longer necessary